### PR TITLE
[RFR][1LP] Latest build fixes

### PR DESCRIPTION
**Pull request by lcouzens**
Fixes to tests that make use of appliance_console on latest 5.9.3.1.

### CODE REVIEW

**Comment by lkhomenk** [comment]
QUESTION: Are we supposed to run test against any other 5.9 except latest? or that is valid for migration tests only?

**Comment by dajoRH** [comment]
Lint report for commit 76f7297ef4c3818c0a0f38b9994d191d9bac590a:

`cfme/fixtures/cli.py`:
- :heavy_exclamation_mark: Line 98:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by quarckster** [comment]
Test for 5.9 failed

**Comment by rlbabyuk** [comment]
Please check PRT results, some of the tests need small fixes

File: cfme/scripting/setup_env.py [hunk 1]
```diff
@@ -92,7 +92,12 @@ def setup_ha_env(cfme_version, provider_type, provider, lease, desc):
     ip2 = apps[2].hostname
     opt = '5' if cfme_version >= "5.8" else '8'
     rep = '6' if cfme_version >= "5.8" else '9'
-    mon = '9' if cfme_version >= "5.8" else '12'
+    if cfme_version < '5.8':
```
Comment (psav) [inline cfme/scripting/setup_env.py]: **REQUIRED:** Please use this instead, it makes the logic of version picking more consistent to read, even if slightly less efficient.

```python
from widgetastic.utils import VersionPick, Version
mon = VersionPick({
    Version.lowest(): '12',
    '5.8': '9',
    '5.9.3': '8'
}).pick(cfme_version)
```

File: cfme/tests/cli/test_appliance_cli.py [hunk 2]
```diff
@@ -195,20 +182,20 @@ def test_appliance_console_cli_ha_crud(unconfigured_appliances, app_creds):
         'vmdb_production', apps[1].unpartitioned_disks[0]
     )
     # Configure automatic failover on EVM appliance
-    command_set = ('ap', '', '9', '1', '')
+    command_set = ('ap', '', '8', '1', '')
```
Comment (psav) [inline cfme/tests/cli/test_appliance_cli.py]: **REQUIRED:** 5.8 support is still required here

File: cfme/tests/cli/test_appliance_console.py [hunk 3]
```diff
@@ -216,11 +181,15 @@ def test_appliance_console_ha_crud(unconfigured_appliances, app_creds):
...
+    if apps[0].version > '5.9':
+        command_set = ('ap', '', '6', '2', '2', app0_ip, '', pwd, '', '1', '2', '', '', pwd, pwd,
+                       app0_ip, app1_ip, 'y', TimedCommand('y', 60), '')
+    else:
+        command_set = ('ap', '', '6', '2', '1', '2', '', '', pwd, pwd, app0_ip, app1_ip, 'y',
+                       TimedCommand('y', 60), '')
     apps[1].appliance_console.run_commands(command_set)
     # Configure automatic failover on EVM appliance
-    command_set = ('ap', '', '9', TimedCommand('1', 30), '')
+    command_set = ('ap', '', '8', TimedCommand('1', 30), '')
```
Comment (psav) [inline cfme/tests/cli/test_appliance_console.py]: **REQUIRED:** This too needs to support 5.8

**Pull request merged**